product-details/product-details.css

```diff
@@ -0,0 +1,231 @@
+@import url("https://fonts.googleapis.com/css2?family=Cairo:wght@600&display=swap");
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Bodoni Moda", serif;
```

Comment:
I think using the font [Tenor sans](https://fonts.google.com/specimen/Tenor+Sans?preview.text=ON%20AIR%20%2F%20OFF%20AIR&preview.text_type=custom&sidebar.open=true&selection.family=Bellota+Text:wght@700#standard-styles) would look better and truer to the Figma design. Other than that, everything is looking good. 
Great job!